docs/changelog.rst

```diff
@@ -11,8 +11,38 @@ The full release history can be viewed `at the GitHub perses releases page <http
 0.10.0 - Release
 ----------------
 
+New command line interface (CLI), many enhancements for the API (specially the ``PointMutationExecutor``) and improved testing.
+
+Bugfixes
+^^^^^^^^
+- Bug in geometry engine when could not generate a successful topology -- fixed by not trying to access it when its ``None``. (`#855 <https://github.com/choderalab/perses/pull/855>`_)
+- Bug generated by changes upstream in the ``openmm`` package -- fixed by adding more padding to the simulation box, it is now 1.1 nm. Openmm issue `#3502 <https://github.com/openmm/openmm/issues/3502>`_. Perses issue `#949 <https://github.com/choderalab/perses/issues/949>`_ (`#953 <https://github.com/choderalab/perses/pull/953>`_)
+- Fixed energy bookkeeping in test of ``HybridTopologyFactory`` when a ring amino acid is involved in transformation. (`#969 <https://github.com/choderalab/perses/pull/969>`_)
+- Avoid changing the global context cache behavior on module imports. Issue `#968 <https://github.com/choderalab/perses/issues/968>`_ (`#977 <https://github.com/choderalab/perses/pull/977>`_).
+- Benchmark free energy plots now shift data to experimental mean. (`#981 <https://github.com/choderalab/perses/pull/981>`_)
+- Recovered logging capabilities respecting the ``LOGLEVEL`` environment variable. Issue `#1018 <https://github.com/choderalab/perses/issues/1018>`_ (`#1032 <https://github.com/choderalab/perses/pull/1032>`_).
+
+
 Enhancements
-------------
+^^^^^^^^^^^^
+- Improved continuous integration (CI) performance. (`#961 <https://github.com/choderalab/perses/pull/961>`_)
+- ``PointMutationExecutor`` now accepts both solute and solvated PDBs. (`#967 <https://github.com/choderalab/perses/pull/967>`_)
+- Tests and examples are now using ``openff-2.0.0`` force field instead of ``openff-1.0.0``. (`#971 <https://github.com/choderalab/perses/pull/971>`_)
+- Use names for fetching the force components of a system, avoiding issues with indices reordering upstream in ``openmm``. Issue `#993 <https://github.com/choderalab/perses/issues/993>`_ (`#976 <https://github.com/choderalab/perses/pull/976>`_ and `#1007 <https://github.com/choderalab/perses/pull/1007>`_)
+- Increase stability of simulations by decreasing the default hydrogen mass to 3 amu. Issue `#982 <https://github.com/choderalab/perses/issues/982>`_ (`#983 <https://github.com/choderalab/perses/pull/983>`_).
+- Improved CI tests on both CPU and GPU (`#985 <https://github.com/choderalab/perses/pull/985>`_ `#989 <https://github.com/choderalab/perses/pull/989>`_ `#1012 <https://github.com/choderalab/perses/pull/1012>`_)
+- ``PointMutationExecutor`` now raises an error if an invalid phase is specified (`#991 <https://github.com/choderalab/perses/pull/991>`_).
```

Comment:
I think we should move this bullet point to the "Bug Fixes" section, since this PR introduces the phase check, but a later PR (#1024) actually removes it. The primary changes that this PR introduces are actually 1) skipping counterion for the vacuum phase and 2) fixing a typo in one of the tests. 

Maybe something like:
```
- Skip introduction of counterion for charge changing mutations in vacuum and fix typo in the phase name in `test_resume_protein_mutation_no_checkpoint` (`#991 <https://github.com/choderalab/perses/pull/991>`_).
```